src/styles/blocks/stars.scss

```
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

```suggestion
.stars {
  display: flex;

 &__star {
 ........
```
you should use nesting

---

thanks